src/bin/vip-whoami.ts

```diff
@@ -6,6 +6,9 @@ import command from '../lib/cli/command';
 import * as exit from '../lib/cli/exit';
 import { trackEvent } from '../lib/tracker';
 
+const baseUsage = 'vip whoami';
+const exampleUsage = 'vip @example-app.develop whoami';
```

Comment:
`whoami` is one of the very few commands that does not require an app alias. 
I suggest that we update the command example to instead be the simpler form:
`vip whoami`
I just tested this command with a non VIP-team user, and I think it would be helpful for the example to include the whole shebang:
```
$ vip whoami
- Howdy user@example.com!
- Your user ID is 1234
```
I am trying my best to ignore the "Howdy" and fight the urge to let this scope creep....

Reply:
That makes sense - I added the exception about application.env aliases to the style guide.

Reply:
Perfect. Thank you!